### ISSUE TITLE
Close Window Edge Click

### ISSUE THREAD

**Issue by LiohMoeller** [mate-desktop/marco#562]
#### Expected behaviour
I have a setup with only one bottom panel (LinuxMint default). When I maximize a window, I expect that i can just push the mouse to the upper right corner and left-click to close it. This works for the Panel but not for the windowmanager

#### Actual behaviour
I have to click exactly on the close button.

#### Steps to reproduce the behaviour
Maximize a window, move the mouse to the most upper right corner and left-click (with one bottom panel setup). The window should close but it does not.

#### Linux Distribution
Linux Mint 19.2

**Comment by lukefromdc** [comment]
Are you using MATE and not Cinnamon? I've never heard of such a behavior in marco or in compiz, and it is common practice instead to have the application icon in the top left corner of the decoration(the org>mate>marco>general>button-layout gsettings value sets this and the min/maz/close button layout). Clicking that icon brings up a menu of window actions including minimize, maximize(or unmaximize) etc, with close at the bottom of the menu.

**Comment by LiohMoeller** [comment]
I am not sure if I have expressed myself clearly enough, so I have made a little screen video showing the behavior:

https://cloud.operationtulip.com/s/XSGm3NF5reKmF5A

As you see, when I move the mouse to the upmost right corner and left-click the window does not close. I have to hit the close button more or less exactly to close the window.

I have tested the behavior on Trisquel Mate, where it does not occur. So it's either specific to the Mate Version included in Mint 19.2 or to the theme. I have tested all themes included in lm Mate, with the same result.

**Comment by lukefromdc** [comment]
I don't think that was evwr an intended feature in marco/metacity(at least prior to the fork if ever),
if you want such a feature in a mate desktop it would come from the window manager. In MATE
you can replace marco with almost any other Xorg window manager. Find one witg that feature and
that you otherwise like and install it and you have this.

**Comment by LiohMoeller** [comment]
I have tried to switch e.g. to compiz or all other options offered in LinuxMint, and it's still the same. I wonder why it works on trisquel, which uses also Macro (GPU Redering). I guess it might somehow be related to the theme, because the close button on Trisquel is very near to the window borders (which is not the case with the MintX theme).

**Comment by lukefromdc** [comment]
A close button right at the corner could certainly do this, if that is the behavior you prefer putting the close button all the way to the left and editing your metacity theme to put it right at the corner with no padding will do it.
